app/services/katello/rhsm_fact_parser.rb

```diff
@@ -146,8 +143,6 @@ def get_rhsm_mac(interface)
     end
 
     def distribution_to_puppet_os(name)
-      return REDHAT_ATOMIC_HOST_OS if name == REDHAT_ATOMIC_HOST_DISTRO_NAME
-
       name = name.downcase
```

Comment:
I noticed this block could be cleaner using a case statement: https://github.com/theforeman/foreman/pull/9838